iosApp/iosApp/ComponentViews/WithRealtimeIndicator.swift

```diff
@@ -12,37 +12,43 @@ struct WithRealtimeIndicator: View {
     private static let subjectSpacing: CGFloat = 4
     @ScaledMetric private var iconSize: CGFloat = 12
     let prediction: any View
+    let visible: Bool
 
-    init(_ prediction: () -> any View) {
+    init(_ prediction: () -> any View, visible: Bool = true) {
```

Comment:
Yeah, I did it this way because conditionally applying modifiers is annoying and verbose, you either need to set the view to variable then use an if statement to apply a modifier, or create a new conditional modifier that can [cause problems](https://www.objc.io/blog/2021/08/24/conditional-view-modifiers/) with animations. Would it make more sense to do the inverse and have `hideIndicator`?